NOTICE

```diff
@@ -1 +1,32 @@
 Copyright 2020-2023, Constellation Network Inc.
+
+This product includes software developed at
+Constellation Network Inc. (https://constellationnetwork.io/).
+
+The below notice derives from: https://github.com/apache/spark/blob/master/NOTICE
+and originates from The Apache Software Foundation (http://www.apache.org/).
+It has been modified to include descriptions of libraries providing crypto
+functionality within this project.
+
+Export Control Notice
+---------------------
+
+This distribution includes cryptographic software. The country in which you currently reside may have
+restrictions on the import, possession, use, and/or re-export to another country, of encryption software.
+BEFORE using any encryption software, please check your country's laws, regulations and policies concerning
+the import, possession, or use, and re-export of encryption software, to see if this is permitted. See
+<http://www.wassenaar.org/> for more information.
+
+The U.S. Government Department of Commerce, Bureau of Industry and Security (BIS), has classified this
+software as Export Commodity Control Number (ECCN) 5D002.C.1, which includes information security software
+using or performing cryptographic functions with asymmetric algorithms. The form and manner of this Apache
+Software Foundation distribution makes it eligible for export under the License Exception ENC Technology
+Software Unrestricted (TSU) exception (see the BIS Export Administration Regulations, Section 740.13) for
+both object code and source code.
+
+The following provides more details on the included cryptographic software:
+
+This software uses Bouncy Castle (https://www.bouncycastle.org/) for digital signatures and validation,
+certificates and certificate validation, key generation and management, and secure random number generation;
+and Java Security API (https://docs.oracle.com/en/java/javase/11/security/java-security-overview1.html)
+for message digests.
```

Comment:
I think it should be a part of a separate commit/PR. Since the title is "fix: remove guava" it should only remove `guava` dependency.

I would put it to separate commit/PR called `docs: add NOTICE` or sth.